tasks/Task.cpp

```diff
@@ -137,6 +137,8 @@ void Task::processIO()
         filtered_rbs.time = time;
         filtered_rbs.angular_velocity =
             filterAngularVelocity(update.rbs.angular_velocity);
+        filtered_rbs.targetFrame = _imu_frame.get();
+        filtered_rbs.sourceFrame = _world_frame.get();
```

Comment:
A RBS is the state of the source in the target. These frames are inverted (the imu outputs the pose of the imu in the world)